contracts/container/contract.go

```diff
@@ -491,6 +494,67 @@ func List(owner []byte) [][]byte {
 	return list
 }
 
+// AddNextEpochNodes adds container members for the next epoch. Changes must be
+// committed with [CommitContainerListUpdate]. Without submitting, it is a no-op
+// operation. Results of the call operation can be received via [Nodes]. This
+// method must be called only when a container list is changed, otherwise
+// nothing should be done.
+// Call must be signed by the Alphabet nodes.
+func AddNextEpochNodes(cID []byte, nodes [][]byte) {
+	ctx := storage.GetContext()
+	multiaddr := common.AlphabetAddress()
+	common.CheckAlphabetWitness(multiaddr)
+
+	for _, node := range nodes {
+		pk := node[2 : 2+interop.PublicKeyCompressedLen] // public key in V2 marshaled node info
+		key := []byte{nextEpochNodesPrefix}
+		key = append(key, cID...)
+		key = append(key, pk...)
```

Comment:
1+32+33 -> looks like an overflow of our key length limit to me (64).

Reply:
We're risking collisions then and we can't easily extract the public key from the DB key.

Reply:
We need public keys to be accessible for #413, but then the key doesn't mean much and we can use `prefix + cid + counter` or something like that.

Reply:
just a question: is a collision (2 bytes less than 33) too scary to complicate things?

> We need public keys to be accessible for #413

pk can be striped on a key suffix and a value. weird but would works and no additional logic with counters and its handling

anyway will fix somehow

Reply:
Probability is low, but the key doesn't make much sense in its current form, it's better to have some meaningful component there.